### ISSUE TITLE
Сообщать, в модуле объекта обработки для функции ПолучитьМакет, если макет обработки не существует.

### ISSUE THREAD

**Issue by DmitryShvaika** [1C-Company/1c-edt-issues#1274]
### Описание проблемы

В модуле объекта обработки размещен текст:

```
// @strict-types -

#Если Сервер Или ТолстыйКлиентОбычноеПриложение Или ВнешнееСоединение Тогда

#Область ПрограммныйИнтерфейс

// Некая процедура.
//
Процедура НекаяПроцедура() Экспорт
	
	НеСуществующийМакет = ПолучитьМакет("НеСуществующийМакет");
	НеСуществующийМакет = НеСуществующийМакет;
	
КонецПроцедуры

#КонецОбласти

#КонецЕсли

```

При этом ошибка для _ПолучитьМакет("НеСуществующийМакет");_ не выдается, хотя макетов у обработки вообще нет.

### Описание решения проблемы

Предлагаю выдавать ошибку, что макет не существует.

### Дополнительная информация

[1Cv8.zip](https://github.com/1C-Company/1c-edt-issues/files/13804571/1Cv8.zip)
выгрузка информационной базы с тестовым примером

**Comment by marmyshev** [comment]
Мне казалось что была такая проверка в ЕДТ.

Когда макеты есть так же должна работать подсказка ввода в строковом литерале.

**Comment by LesN1ck** [comment]
> Мне казалось что была такая проверка в ЕДТ.
> 
> Когда макеты есть так же должна работать подсказка ввода в строковом литерале.

Подсказка ввода работает, регулярно пользуюсь.

**Comment by DmitryShvaika** [comment]
т.е. надо регистрировать как ошибку?

**Comment by LesN1ck** [comment]
> т.е. надо регистрировать как ошибку?

Нет, речь про разные вещи. Контекстная подсказка работает - да (ошибку имеет смысл регистрировать, только если она у вас не работает). Валидатор не проверяет параметр метода "ПолучитьМакет" на несуществующие макеты - тоже да (и об этом как раз ваше пожелание).
